[pytorch][gh1] Fix failing test

bigfootjon opened the pull request:
Summary: See tasks, these tests are broken in fbsource

Test Plan: `buck test fbcode//caffe2/.github/scripts:`

Differential Revision: D37182246

malfet reviewed (approved):
LGTM, but can we set the environment only if it is not defined?

bigfootjon committed:
[pytorch][gh1] Fix failing test (#79632)

Summary:
Pull Request resolved: https://github.com/pytorch/pytorch/pull/79632

See tasks, some tests are broken in fbsource

Test Plan: `buck test fbcode//caffe2/.github/scripts:`

Reviewed By: jailby

Differential Revision: D37182246

fbshipit-source-id: 3cfdc3eb30739f92f3757f0ae19713666c1555c4

seemethere commented on .github/scripts/test_trymerge.py:
```diff
@@ -17,6 +17,9 @@
 from unittest import TestCase, main, mock
 from urllib.error import HTTPError
 
+if 'GIT_REMOTE_URL' not in os.environ:
+    os.environ['GIT_REMOTE_URL'] = "https://github.com/pytorch/pytorch"
```
nit, this can probably be done in one line, but overall lgtm

```suggestion
os.environ['GIT_REMOTE_URL'] = os.environ.get(GIT_REMOTE_URL, "https://github.com/pytorch/pytorch")
```

bigfootjon replied:
I'm meh on this, it still requires hardcoding "GIT_REMOTE_URL" twice, and is less easy to read IMO
